idaes/models/control/controller.py

```diff
@@ -92,27 +116,27 @@ class PIDControllerData(UnitModelBlockData):
             description="Type of bounds to apply to the manipulated variable (mv)).",
             doc=(
                 """Type of bounds to apply to the manipulated variable output. If,
-bounds are applied, the model parameters **mv_lb** and **mv_ub** set the bounds.
-The **default** is ControllerMVBoundType.NONE. See the controller documentation
-for details on the mathematical formulation. The options are:
-**ControllerMVBoundType.NONE** no bounds, **ControllerMVBoundType.SMOOTH_BOUND**
-smoothed mv = min(max(mv_unbound, ub), lb), and **ControllerMVBoundType.LOGISTIC**
-logistic function to enforce bounds.
-"""
+                bounds are applied, the model parameters **mv_lb** and **mv_ub** set the bounds.
+                The **default** is ControllerMVBoundType.NONE. See the controller documentation
+                for details on the mathematical formulation. The options are:
+                **ControllerMVBoundType.NONE** no bounds, **ControllerMVBoundType.SMOOTH_BOUND**
+                smoothed mv = min(max(mv_unbound, ub), lb), and **ControllerMVBoundType.LOGISTIC**
+                logistic function to enforce bounds.
+                """
```

Comment:
You may want to look at the generated docs on this one.  I know it's ugly the original way, but this will include the indentation in the string.